nf_core/pipeline-template/{{cookiecutter.name_noslash}}/README.md

```diff
@@ -21,14 +23,17 @@ ii. Install one of [`docker`](https://docs.docker.com/engine/installation/), [`s
 iii. Download the pipeline and test it on a minimal dataset with a single command
 
 ```bash
-nextflow run {{ cookiecutter.name }} -profile test,<docker/singularity/conda>
+nextflow run {{ cookiecutter.name }} -profile test,<docker/singularity/conda/institute>
 ```
 
+> Please check [nf-core/configs](https://github.com/nf-core/configs#documentation) to see if a custom config file to run nf-core pipelines already exists for your Institute. If so, you can simply use `-profile institute` in your command. This will enable either `docker` or `singularity` and set the appropriate execution settings for your local compute environment.
```

Comment:
It's here @drpatelh

Reply:
Yes, good point 👍 I wrote that 😅 but the explanation beforehand hopefully makes that clearer whereas we cant put that in the parameter section...

Maybe you can update both of those in the next release?